fem/fe.hpp

```diff
@@ -632,6 +672,13 @@ class NodalFiniteElement : public ScalarFiniteElement
                        DenseMatrix &curl) const;
 
 public:
+   /** @brief Construct FiniteElement with given
```

Comment:
Should use the class name rather than parent class name.

Also, it might be good to include a short description of the class at the beginning of the class declaration